wiki/Rules/Content_Usage_Guidelines/en.md

```diff
@@ -0,0 +1,39 @@
+# Content Usage Guidelines
+
+A user should have permission from relevant copyright holders for audio/visual/gameplay assets included in their beatmap submissions. Without permission, the beatmap is at risk of being removed and the creator is at risk of being denied access to the submission system as per the [osu! copyright policy](/wiki/legal/copyright).
+
+To ensure a user's account remains in good standing, it is recommended to create beatmaps using assets that are any of the following:
+
+- legally cleared for use in osu!
+- permitted for use by their creator(s)
+- copyright-free
+
+## Audio
+
+osu! provides its userbase a library of licensed songs via the [Featured Artist](/wiki/Featured_Artists) program. Each song on this listing is free for use within osu! and has no risk of removal.
+
+When looking for songs outside of osu!'s featured artist catalog, check whether or not its creator has explicitly stated usage terms. This information can often be found in description sections of an artist's music release platforms (e.g. Bandcamp, Soundcloud, YouTube) or their personal website. If a song is licensed under a [Creative Commons License](https://creativecommons.org/licenses/by-nc-sa/3.0/) or is available for non-commercial use, it is safe to create and submit a beatmap for it.
+
+If you want to create a beatmap for a song and its usage rights are not clear, it is recommended to reach out to its artist for permission. Most artists have contact methods labeled on their music release platforms, personal websites, and social media.
```

Comment:
```suggestion
If you want to create a beatmap for a song and its usage rights are not clear, it is recommended to reach out to its artist for permission. Most artists have contact methods labeled on their music release platforms, personal websites, and social media.

This may feel like a large hurdle, but we've seen mappers have pretty high success and get artists' approval just by taking the courage to ask before using content without permission. It may help to mention that osu! is a free no-ad game, and you can link them to our [licensing](/wiki/Legal/Music_Licensing) page to give them a better idea of how content is consumed.
```

Mention the licensing page and try and encourage mappers to reach out more.